configure

```diff
@@ -32,7 +32,8 @@ ninja
   .description('$command')
 
 let browserify = `browserify --extension='.jsx' --transform [ babelify \
-    --presets [ @babel/react @babel/env ] ]`
+    --presets [ @babel/react @babel/env ] ] --transform [ stringify \
+    --extensions [.html] ] `
```

Comment:
Why does html need to run through browserify?

Reply:
Yeah, weird, huh? I need to get at the HTML file that IBOM uses to include it inside a React component. (I also need to edit the HTML a bit, but that's not important for this point.) That means I need to be able to `require` an HTML file, which means browserify needs to know about it. I think. But that's all a bit of a hack, so we should probably talk about it in more detail once I've got closer to something I think is working.

Reply:
I have used `html-to-react` and `htmltojsx` for these sorts of problems on Kitspace before.

Reply:
OK, it's useful to know about those. I'll switch over to using something like that. (Did I mention at any point that I know very little about React?) I also want to move that HTML processing completely to build time anyway, since some of it still happens on each IBOM page load, which is a bit stupid. (All down to me trying to find a way to do things with minimal changes to IBOM itself.)